src/h5web/dimension-mapper/SlicingSlider.tsx

```diff
@@ -30,6 +30,7 @@ function SlicingSlider(props: Props): ReactElement {
         // https://github.com/zillow/react-slider/issues/172
         key={`${mapperState.includes('y')}`}
         className={styles.slider}
+        ariaLabel="Dimension slider"
```

Comment:
You could add the number `dimension` in the label (i.e. `Dimension {dimension} slider`)

Reply:
The dimension index, yes, but not the size of the current dimension... and I think `dimension` is the latter 😢 -- I'd have to pass an extra prop to get the index, which seems overkill for this.